chain/client/src/stateless_validation/state_witness_distribution_actor.rs

```diff
@@ -0,0 +1,59 @@
+use actix::Actor;
+use near_async::messaging::Sender;
+use near_async::{MultiSend, MultiSendMessage, MultiSenderFrom};
+use near_network::types::PeerManagerAdapter;
+use near_o11y::{handler_debug_span, WithSpanContext};
+use near_performance_metrics_macros::perf;
+use near_primitives::stateless_validation::SignedEncodedChunkStateWitness;
+use near_primitives::types::AccountId;
+
+use super::state_witness_distribution_actions::StateWitnessDistributionActions;
+
+pub struct StateWitnessDistributionActor {
+    pub actions: StateWitnessDistributionActions,
+}
+
+impl StateWitnessDistributionActor {
```

Comment:
Might (will) change the name from `StateWitnessDistributionActor` to `StateWitnessActor` in a future PR.

Reply:
I would personally prefer shorter name, as it is not clear whether we will need it to do more operations or not at this point.

Reply:
On it! Upcoming PR!

Reply:
https://github.com/near/nearcore/pull/11089